userController.js

```diff
@@ -0,0 +1,26 @@
+import { user } from '../models';
+ import { downloadResource } from '../util';
+
+ const controller = {};
+
+ controller.download = async (req, res) => {
+  const fields = [
+    {
+      label: 'Student ID',
+      value: 'googleID'
```

Comment:
By Student ID we mean the email id of the student needs to be sent.

Reply:
Ok. I'll do the changes

Reply:
Changes done